src/main/java/io/fabric8/maven/docker/service/RegistryService.java

```diff
@@ -45,6 +45,11 @@ public void pushImages(Collection<ImageConfiguration> imageConfigs,
                            int retries, RegistryConfig registryConfig, boolean skipTag) throws DockerAccessException, MojoExecutionException {
         for (ImageConfiguration imageConfig : imageConfigs) {
             BuildImageConfiguration buildConfig = imageConfig.getBuildConfiguration();
+            if (buildConfig.skipPush()) {
```

Comment:
I know there is a test missing in `RegistryServiceTest` for pushImages, but it would be cool if you could add one to test this change, too (then the Sonar check would be happy, too). It shouldn't be hard to add, as it should work the same as the pull test, too.